.circleci/smoke_test/docker/build_and_push.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
```

Comment:
Par our internal conversation we should probably move the comment in Dockerfile here and update it so that it does not mention AWS ECR but Docker hub.